### ISSUE TITLE
[Bug] Firmware version cannot be parsed

### ISSUE THREAD

**Issue by arturpragacz** [Vaskivskyi/ha-asusrouter#873]
### The problem

```
WARNING (MainThread) [asusrouter.modules.firmware] Firmware version cannot be parsed. Please report this. The original FW string is: `__`
```

### Your device model

RT-N66U

### Firmware type

Stock

### Firmware version

3.0.0.4.382_52711

### Integration version with the issue

0.34.0

### Method of the integration installation

HACS

### What version and type of Home Assistant installation do you use

Core 2024.9.0; Operating System 13.1

**Comment by Vaskivskyi** [comment]
Thanks for your report, @arturpragacz,

The warning will be removed in the next update (~ several days) and is already fixed in the backend library

**Comment by arturpragacz** [comment]
@Vaskivskyi Can you bump the integration to fix this?

**Comment by Vaskivskyi** [comment]
I am really sorry for this long delay. I was completely sure, that the fix was already released. Please, check whether it is fixed in `0.34.1`
